testing-modules/selenium-2/src/test/java/com/baeldung/selenium/find/CheckIfElementPresentManualTest.java

```diff
@@ -0,0 +1,62 @@
+package com.baeldung.selenium.find;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+import java.util.List;
+
+import static org.testng.Assert.assertTrue;
+
+public class CheckIfElementPresentManualTest {
```

Comment:
Why are we using Testng?

Reply:
replaced for JUnit

Reply:
Thanks. Please remove `public` keyword from `class` and functions, since JUnit5 uses package private by default.